zeppelin-plugins/launcher/yarn/pom.xml

```diff
@@ -49,6 +49,11 @@
       <artifactId>hadoop-yarn-api</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
```

Comment:
Based on the Hadoop profile in zeppelin-zengine, the Hadoop lib may or may not be present.
We should also put Hadoop profiles in this `pom.xml` and explicitly set this dependency to `provided`.

Reply:
still valid